Purchases/Networking/Operations/NetworkOperation.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  NetworkOperation.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class CacheableNetworkOperation: NetworkOperation, Cachable {
+
+    var key: String { "\(Self.description()) \(individualizedCacheKeyPart)" }
+
+    let individualizedCacheKeyPart: String
+
+    init(configuration: NetworkConfiguration, individualizedCacheKeyPart: String) {
```

Comment:
`individualizedCacheKeyPart` is the part of the cacheKey that makes it unique from other operations of the same type. Like if you posted receipts two times in a row you'd have 2 operations. The cache key would be `PostOperation + individualizedCacheKeyPart` where `individualizedCacheKeyPart` is whatever you determine to be unique.

Reply:
could we add that^^ awesome explanation into a comment?

Reply:
🤦‍♂️ yup

Reply:
✅